ControlFlow/Program.cs

```diff
@@ -45,40 +62,89 @@ public static string LandscapeOrPortrait(int width, int height)
 
+            if (speedCar <= speedLimit)
+            {
+                return Convert.ToString("Okay");
+            }
+            else
+            {
+                for (int tooFast = 5; tooFast <= speedDifference; tooFast += 1)
+                {
+                    if (tooFast % 5 == 0)
+                    {
+                        demeritScore++;
+                    }
+                }
+            }
+            if (demeritScore > 12)
+            {
+                return Convert.ToString("License Suspended");
+            }
+            else
+            {
+                return Convert.ToString(demeritScore);
+            }
+        }
         // Part 2, Control Flow. Come back to this section after completing the For/Foreach/While loops.
 
         // 2.1 - Write a method to count how many integers between two numbers are divisible by a divider with
         // no remainder. Return the count. For example, if minNumber = 1 and maxNumber = 6, and the divider is 3 then
         // output should be 2. Don't Worry about input validation.
         public static int DivisibleByNumber(int minNumber, int maxNumber, int divider)
         {
-            return default;
+            int divisibles = 0;
+
+            for (int i = minNumber; i <= maxNumber; i++)
+                if (i % divider == 0)
+                    divisibles++;
+
+            return divisibles;
         }
 
         // 2.2 - Write a method to calculate the sum of all the integers and return it. For example if
         // the program enters (2, 3, 5, 1) then the return should be 11. Don't Worry about input validation.
         public static int SumIntegers(params int[] numbers)
         {
-            return default;
+            int sum = 0;
+
+            foreach (int integer in numbers)
+            {
+                sum += integer;
+            }
+            return sum;
         }
 
         // 2.3 - Write a method to compute the factorial of an integer return it. For example, if the
         // user enters 5, the program should calculate 5 x 4 x 3 x 2 x 1 and return it as 120. Make sure
         // that 0! = 1.
         public static int FindFactorial(int number)
         {
-            return default;
+            int product = 1;
+
+            while (number != 0)
+            {
+                product = product * number;
+                number = number - 1;
+            }
+            return product;
         }
 
         // 2.4 - Write a method that takes a series of integers separated by commas. Then find the
         // maximum of the numbers and return it. For example, if the numbers are “5, 3, 8, 1, 4",
         // the program should return 8.
         public static int MaximumNumber(params int[] numbers)
         {
-            return default;
+            int max = numbers[0];
+
+            foreach (int digit in numbers)
+            {
+                if (digit > max)
+                    max = digit;
+            }
```

Comment:
Consider using a for loop that skips the first element instead. This is a very minor issue but otherwise this document is great! Keep it up 👍 

So instead do

``
for (int digit = 1; digit < numbers.length ; digit++) 
``

Because you don' need to iterate through digit[0] like you are in the foreach. Otherwise this looks great!